assets/loadtest/cluster/aws/cluster.yaml

```diff
@@ -4,6 +4,8 @@ metadata:
   name:
   region:
   version: "1.29"
+  tags:
+    teleport.dev/creator:
```

Comment:
if this works great, but could we put "creator: loadtest" or add a terraform variable for creator email?